api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -770,25 +783,25 @@ private void enableVitals() {
     }
 
     private void enableConsult() {
-        if (config.getCountry().equals(ConfigDescriptor.Country.PERU) ){
+        if (config.getCountry().equals(ConfigDescriptor.Country.PERU)) {
```

Comment:
Do we want to remove Peru?

Reply:
This was just a stuff the auto-formatter fixed... but, yeah, we should clean up up the Peru stuff... feel free to ticket? :)